doc/lightning-delpay.7.md

```diff
@@ -0,0 +1,84 @@
+lightning-delpay -- Command for removing a completed or failed payment
+============================================================
+
+SYNOPSIS
+--------
+
+**delpay** *payment\_hash* \[status\]
+
+DESCRIPTION
+-----------
+
+The **delpay** RPC command removes a payment as given in **listsendpays** or **listpays** with a complete or failed
+status. Deleting a `pending` payment is an error.
+
+- *payment\_hash*: The unique identifier of a payment. Find it, you can run **listpays** or **listsendpays**;
+- *status*: Expected status of the payment. Valid values are *complete* or *failed*. 
+Only deletes if the payment status matches. If not specified, defaults to *complete*.
```

Comment:
Shouldn't an unspecified value for `status` mean we don't care, and either `complete` or `failed` will be removed?

Reply:
Since `delpay` is supposed to mirror `delinvoice`, which has the `*status*` as a required arg, maybe it should be a required arg here as well?

Reply:
Yep I agree with this point! thanks for clarifying!